mod_reforged/hooks/entity/world/location.nut

```diff
@@ -20,13 +20,27 @@
 	q.getTooltip = @(__original) function()
 	{
 		local ret = __original();
+
+		local worth = 0;
+		foreach (t in this.m.Troops)
+		{
+			foreach (troop in ::Const.World.Spawn.Troops)
+			{
+				if (troop.ID == t.ID && troop.Script == t.Script)
+				{
+					worth += troop.Cost;
+					break;
+				}
+			}
+		}
+
 		if (!this.isHiddenToPlayer() && this.m.Troops.len() != 0 && this.getFaction() != 0)
 		{
 			ret.push({
 				id = 100,
 				type = "text",
 				icon = "ui/icons/icon_contract_swords.png",
-				text = format("Strength: %s / %s", ::MSU.Text.colorGreen(::World.State.getPlayer().getStrength()), ::MSU.Text.colorRed(this.getStrength()))
+				text = format("Strength: %s / %s", ::MSU.Text.colorGreen(::World.State.getPlayer().getStrength()), ::MSU.Text.colorRed(worth))
```

Comment:
`Strength` is now misleading. Only the first value is strength (player party strength)

I suggest changing it into
`Cost: %s (Player Strength: %s)`

After all, we are viewing a different party so the assumption would be that the first/main value is about that party and not us

Reply:
Yeah I tried at first to write it as `Player Strength: %s / Target Resources: %s` or something like that but it becomes too long and goes on two lines and I want to avoid that. So then I went with just this even though it isn't really Strength.

But I agree I don't like this either. So, maybe something like `Cost: %s (Player: %s)` is enough?

Reply:
That ok